packages/api/__tests__/utility.ts

```diff
@@ -33,7 +34,7 @@ class ExampleURLAdapter implements URLAdapter {
     return `https://demo.wepublish.ch/page/${page.id}/${page.slug}`
   }
 
-  getPeeredArticleURL(peer: Peer, article: PublicArticle): string {
+  getPeeredArticleURL(peer: Peer, article: Article): string {
```

Comment:
Shouldn't this be PublicArticle? Since the website only gets PublicArticle and not Article?